scripts/install-app.sh

```
@@ -92,20 +89,20 @@ esac
-    filename="Kata-Agents-${arch}.dmg"
+    filename="Kata-Desktop-${arch}.dmg"
 else
     # Linux only supports x64 currently
     if [ "$arch" != "x64" ]; then
         error "Linux currently only supports x64 architecture. Your architecture: $arch"
     fi
     platform="linux-${arch}"
-    APP_NAME="Kata-Agents-x64.AppImage"
+    APP_NAME="Kata-Desktop-x64.AppImage"
     INSTALL_DIR="$HOME/.local/bin"
     ext="AppImage"
```

Shellcheck (0.11.0)

[warning] ext appears unused. Verify use (or export if used externally).

(SC2034)